controllers/teams.js

```diff
@@ -1058,6 +1060,11 @@ router.get('/:teamId/members', async (req, res, next) => {
 			} else {
 				actions = addButtonTrash(actions);
 			}
+			let userRolesContainsStudent = false;
+			if (FEATURE_STUDENTS_CANT_BE_TEAM_ADMINISTRATOR_ENABLED) {
```

Comment:
This is a little bit difficult. We do not want checks permissions over the roles in future. The preferred way for this is to add new permissions in backend and add this to the roles. "CAN_BE_A_TEAM_ADMIN" and add this over migration the the role in the DB. 
The way should go over the backend and test add this position if the permission exist or not.